testing/scenario_app/ios/Scenarios/ScenariosUITests/iPadGestureTests.m

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <XCTest/XCTest.h>
+
+static const NSInteger kSecondsToWaitForFlutterView = 30;
+
+@interface iPadGestureTests : XCTestCase
+
+@end
+
+@implementation iPadGestureTests
+
+- (void)setUp {
+  [super setUp];
+  self.continueAfterFailure = NO;
+}
+
+#ifdef __IPHONE_15_0
+- (void)testPointerButtons {
+  if (@available(iOS 15, *)) {
```

Comment:
How come this is asking for iOS 15 when the implementation check is for iOS 13.4?

Reply:
While the pointer/trackpad features on iPad came out in iPadOS 13.4, synthesizing those events in XCTest wasn't supported until this year's new Xcode/iPadOS beta releases.

Reply:
You aren't synthesizing any events here though, right?  At the least please add a comment, preferably one that points to the API that is being used that requires ios 15.